src/core/src/engine.rs

```diff
@@ -108,7 +115,9 @@ impl Engine for Poet2Engine {
                                         && block.block_id > chain_head.block_id)
                                 {
                                     info!("Committing {:?}", block);
-                                    service.commit_block(block_id);
+                                    service.commit_block(block_id.clone());
+                                    block_num_id_map.insert(block_num, block_id);
```

Comment:
This is not used anywhere.

Reply:
Will be made use of in the fork resolution logic.